hugegraph-api/src/main/java/com/baidu/hugegraph/api/metrics/MetricsAPI.java

```diff
@@ -61,6 +73,29 @@ public String system() {
     }
 
     @GET
+    @Timed
+    @Path("backend")
+    @Produces(APPLICATION_JSON_WITH_CHARSET)
+    @RolesAllowed("admin")
+    public String backend(@Context GraphManager manager) {
+        Map<String, Map<String, Object>> results = InsertionOrderUtil.newMap();
+        for (String graph : manager.graphs()) {
+            GraphTransaction tx = manager.graph(graph).graphTransaction();
+            Map<String, Object> metrics = InsertionOrderUtil.newMap();
+            metrics.put(BackendMetrics.BACKEND, tx.store().provider().type());
+            try {
+                metrics.putAll(tx.metadata(null, "metrics"));
+            } catch (UnsupportedOperationException e) {
+                metrics.put(BackendMetrics.EXCEPTION, e.getMessage());
+                LOG.warn("Failed to get 'metrics' metadata", e);
```

Comment:
LOG.debug is ok since it's an expected exception and will print each time
also update message: "Failed to get backend metrics"